domain-models-runtime-it/ramls/myitems.raml

```diff
@@ -44,6 +44,13 @@ resourceTypes:
       collection-item:
         exampleItem: !include examples/myitem.json
         schema: myitem
+    patch:
+      body:
+        application/json:
+          type: myitem
+      responses:
+        204:
+          description: Accepted
```

Comment:
I noticed there is no PUT or DELETE defined here. I am curious why adding PATCH would make a real difference?

Reply:
For PUT and DELETE respond common ResourceType from raml-util https://github.com/folio-org/raml-module-builder/blob/c3398f318548f22f460ee0840279fd07b5ce817a/domain-models-runtime-it/ramls/myitems.raml#L23

Reply:
I see. Should Patch be added to item-collection.raml then?

Reply:
There are other tests for PUT and DELETE.
DELETE does not have a body and therefore doesn't need the body handler, PUT already has the body handler, and this PR adds PATCH for the body handler: https://github.com/folio-org/raml-module-builder/blob/960bc78bc74079ae2e1a339063ac6e06b4d06301/domain-models-runtime/src/main/java/org/folio/rest/RestVerticle.java#L251-L265
If you comment out the line
`router.patch().handler(handler);`
the unit test MyitemsApiTest.patch will fail because the body is null:
```
java.lang.IllegalArgumentException: HV000116: The object to be validated must not be null.
        at org.hibernate.validator.internal.util.Contracts.assertNotNull(Contracts.java:41)
        at org.hibernate.validator.internal.engine.ValidatorImpl.validate(ValidatorImpl.java:190)
        at org.folio.rest.RestVerticle.isValidRequest(RestVerticle.java:1444)
```

Reply:
Thanks. So if I understand the PR correctly, it is about `router.patch().handler(handler);`. If a module wants to use PATCH, it has to define it just like `myitems.raml`. There is no plan to make PATCH into a reusable raml file like `item-collection.raml` yet.